Framework/API/test/WorkspaceOpOverloadsTest.h

```diff
@@ -204,6 +204,14 @@ class WorkspaceOpOverloadsTest : public CxxTest::TestSuite {
     TS_ASSERT_EQUALS(ws->readY(1)[0], 1.0)
     TS_ASSERT_EQUALS(ws->readE(0)[0], 1.0)
     TS_ASSERT_EQUALS(ws->readE(1)[0], 1.0)
+
+    // MakeDistribution throws exception if input WS uses point data
```

Comment:
Having smaller separate test should usually be preferred, e.g., `test_makeDistribution_fails_for_point_data()` (again, avoiding a comment by giving meaningful names).